### PR TITLE
v.0.5.25: contact form, country selector: display browser native styling to show selector arrow

### PR DESCRIPTION
**Pull request by iisa**
image

### CODE REVIEW

File: packages/donation-form/src/form-elements/contact-form/contact-form.ts [hunk 1]
```diff
@@ -411,17 +411,14 @@ export class ContactForm extends LitElement {
         }
 
         contact-form #donation-contact-form-countryCodeAlpha2 {
-          width: ${iconFieldWidth};
+          width: calc(100%);
```
Comment (latonv) [inline packages/donation-form/src/form-elements/contact-form/contact-form.ts:414]: Q: Can we just do `100%` without the `calc`?

File: packages/donation-form/src/form-elements/contact-form/contact-form.ts [hunk 2]
```diff
@@ -411,17 +411,14 @@ export class ContactForm extends LitElement {
...
           height: 100%;
           box-sizing: border-box;
           font-weight: bold;
           font-size: ${fieldFontSize};
           font-family: ${fieldFontFamily};
           border: 0;
           background: #fff;
-          -webkit-appearance: none;
-          -moz-appearance: none;
-          appearance: none;
```
Comment (latonv) [inline packages/donation-form/src/form-elements/contact-form/contact-form.ts:424]: 👍

**Review by latonv — APPROVED** [review]
LGTM, no blockers -- just one possible simplification.

File: packages/donation-form/src/form-elements/contact-form/contact-form.ts (hunk 1 above)
Comment (iisa) [reply packages/donation-form/src/form-elements/contact-form/contact-form.ts:414]: maybe?

**Pull request merged**